### ISSUE TITLE
Error: Server error or API not found, plugins/check/index.php says everything ok

### ISSUE THREAD

**Issue by 13am** [electerious/Lychee#633]
### Detailed description of the problem

I've migrated from my old Ubuntu to a new Ubuntu 16.04 server. Having migrated lychee to the new server, I get the "Error: Server error or API not found" message when I try to visit my Lychee installation. The output of plugins/check/index.php is as follows:

Diagnostics
-----------
Warning: Dropbox import not working. No property for dropboxKey
Warning: You may experience problems when uploading a large amount of photos. Take a look in the FAQ for details.
No critical problems found. Lychee should work without problems!

System Information
------------------
Lychee Version:  3.0.6
DB Version:      030003
System:          Linux
PHP Version:     7
MySQL Version:   50715
Imagick:         -
Imagick Active:  1
Imagick Version: -
GD Version:      2.1.1
Plugins:         

### Steps to reproduce the issue

Gp to the lychee front page.

### Output of the diagnostics (Settings => Diagnostics)

Diagnostics
-----------

(plugins/Diagnostics/index.php really doesn't show anything but that single word and the dashes)

### Browser and system

Ubuntu 16.04

**Comment by qligier** [comment]
Anything in your PHP logs?

**Comment by 13am** [comment]
I figured it out: there was nothing in my PHP logs but browsing through the apache logs pointed me to the right direction. I had set up SSL certificates for another virtual host on the same machine. The certificates for the virtual host runnig Lychee were however not correctly set and that messed up https when accessing Lychee. Everything works now. Keep up the amazing work with Lychee, I totally enjoy using it!
